.github/workflows/python-app.yml

```diff
@@ -36,7 +36,7 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        if [ -f deep_quoridor/requirements.txt ]; then pip install -r deep_quoridor/requirements.txt; fi
+        if [ -f deep_quoridor/requirements.txt ]; then pip install -r deep_quoridor/ci_requirements.txt; fi
```

Comment:
```suggestion
        if [ -f deep_quoridor/ci_requirements.txt ]; then pip install -r deep_quoridor/ci_requirements.txt; fi
```

Reply:
I made this change in my second commit, but you caught it before I pushed that :-)